flow/layers/layer_builder.cc

```diff
@@ -0,0 +1,232 @@
+// Copyright 2017 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/flow/layers/backdrop_filter_layer.h"
+#include "flutter/flow/layers/clip_path_layer.h"
+#include "flutter/flow/layers/clip_rect_layer.h"
+#include "flutter/flow/layers/clip_rrect_layer.h"
+#include "flutter/flow/layers/color_filter_layer.h"
+#include "flutter/flow/layers/container_layer.h"
+#include "flutter/flow/layers/layer.h"
+#include "flutter/flow/layers/layer_builder.h"
```

Comment:
The primary header should be first, followed by a blank line, followed by the rest of the headers.  This pattern ensures that each header can build on its own (and doesn't require other headers to be included first).

Reply:
Done.